source/_docs/faq.md

```diff
@@ -141,6 +141,90 @@ You'll need the plugin. [More information on sessions](/docs/wordpress-sessions/
 Yes. See [Public Distributions](/docs/start-state/#public-distributions) for details.
 
 
+## DNS
+
+### What is DNS propagation?
+
+Every DNS record has a **Time To Live** (**TTL**) value, which specifies how long any DNS server should hold that record, before dropping it and asking for a new version from its upstream DNS provider. TTLs are usually set in seconds with a few common ones being `86400` (24 hours),  `43200` (12 hours), `3600` (1 hour), and `500` (5 minutes).
+
+**DNS Propagation** is the time it takes for changes made to DNS records to be reflected across DNS servers globally. A lower TTL value means faster propagation, but it's important to note that it is not a 1:1 ratio. Between your <a href=#where-are-my-dns-records-hosted data-proofer-ignore>authoritative name server</a> and the DNS servers of any particular ISP could be any number of intermediate DNS servers. Each server in that chain will wait for the records it holds to expire before requesting new ones. Because of this, it can take *several times longer* than your record's TTL value to see changes reflected for everyone.
```

Comment:
@rachelwhitton I don't know why this anchor link won't work.

Reply:
@alexfornuto Special characters are stripped from TOC anchors client side (see [jquery.tocify.js#L379](https://github.com/pantheon-systems/documentation/blob/master/source/docs/assets/js/jquery.tocify.js#L379))

`#where-are-my-dns-records-hosted` is reportedly not found because the client side directive to strip the special character is not executable in context of the test